webauthn/helpers/decoded_public_key_to_cryptography.py

```diff
@@ -61,5 +63,15 @@ def decoded_public_key_to_cryptography(
         okp_pub_key = Ed25519PublicKey.from_public_bytes(public_key.x)
 
         return okp_pub_key
+    elif isinstance(public_key, DecodedMLDSAPublicKey):
+        """
+        alg is -48, -49 (ML-DSA)
+        """
+        if public_key.alg != COSEAlgorithmIdentifier.ML_DSA_44 and public_key.alg !=COSEAlgorithmIdentifier.ML_DSA_65:
+            raise UnsupportedPublicKey(
+                f"ML-DSA Public key with unsupported algorithms"
```

Comment:
Let's make this a bit more readable, and also report the unrecognized algorithm ID to help with troubleshooting:
```suggestion
        supported_ml_dsa_algs = (
            COSEAlgorithmIdentifier.ML_DSA_44,
            COSEAlgorithmIdentifier.ML_DSA_65,
        )
        if public_key.alg not in supported_ml_dsa_algs:
            raise UnsupportedPublicKey(
                f"ML-DSA Public key with unsupported algorithm {public_key.alg}"
```